lib/improver/tests/blending/weights/test_ChooseWeightsLinear.py

```diff
@@ -130,14 +131,6 @@ def test_dictionary_key_mismatch(self):
             _ = ChooseWeightsLinear(
                 self.weighting_coord_name, config_dict=self.config_dict)
 
-    def test_dictionary_key_match(self):
```

Comment:
Why was this test removed?

Reply:
It wasn't actually doing anything except checking we didn't trip an error, and @MoseleyS noticed the way I'd written it caused pylint to object.  Since we're passing in dictionaries of the correct format elsewhere, in eg `Test_process`, that functionality is covered and I judged the test could be removed.